spec/classes/plugin/nginx_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe 'letsencrypt::plugin::nginx' do
+  on_supported_os.each do |os, facts|
```

Comment:
It's preferred to use `os_facts` to not shadow `facts`
```suggestion
  on_supported_os.each do |os, os_facts|
```